README.old.md

```diff
@@ -1 +1,86 @@
-# calculator
\ No newline at end of file
+<div align="center">
+  <h1><b>Calculator</b></h1>
+
+A calculator app built with React.
+<br>  
+  
+
+## Live Version
+https://flamboyant-clarke-bee74a.netlify.app/
```

Comment:
- [ ] It seems you mistakenly changed the readme filename from `readme.md` to `readme.old.md`. Kindly change the filename to the proper form to enable GitHub to render it on the branch page.